micro-rdk/src/common/ota.rs

```diff
@@ -304,11 +319,19 @@ impl<S: OtaMetadataStorage> OtaService<S> {
             uri = hyper::Uri::from_parts(parts).map_err(|e| OtaError::Other(e.to_string()))?;
         };
 
+        let mut num_tries = 0;
         let (mut sender, conn) = loop {
+            num_tries += 1;
```

Comment:
because of where the increment is, this ends up only trying a maximum of twice rather than thrice?

Reply:
ah you're right. should go at the bottom of the loop. fixed